packages/core/helper-plugin/src/types/custom.d.ts

```diff
@@ -0,0 +1,16 @@
+export {};
+
+declare global {
+  interface Window {
+    strapi: {
+      backendURL: string;
+      isEE: boolean;
+      features: {
+        SSO: 'sso';
```

Comment:
I'm not sure this is accurate @gu-stav recently worked with EE features, he can probably share some insight 👍🏼

Reply:
yeah if you don't need it all, i'd keep it refined for now 👍🏼